CSA.md

```diff
@@ -154,62 +156,66 @@ During the <span class="orderform_link">Subscription Period</span> and for six m
 
-    15.15 **"High Risk Activity"** means any situation where the use or failure of the Cloud Service could be reasonably expected to lead to death, bodily injury, or environmental damage. Examples include full or partial autonomous vehicle technology, medical life-support technology, emergency response services, nuclear facilities operation, and air traffic control.
+    15.15 **"High Risk Activity"** means any situation where the use or failure of the Product could be reasonably expected to lead to death, bodily injury, or environmental damage. Examples include full or partial autonomous vehicle technology, medical life-support technology, emergency response services, nuclear facilities operation, and air traffic control.
 
     15.16 **"Indemnifying Party"** means a party to this Agreement when the party is providing protection for a particular Covered Claim.
 
-    15.17 **"Order Form"** means an order form signed or electronically accepted by the parties that references a Cover Page. Order Forms may include details about the level of access and use granted to the Cloud Service, nature and timing of <span class="orderform_link">Professional Services</span>, extent of <span class="orderform_link">Product Support</span>, or other details about the Cloud Service.
+    15.17 **"Key Terms”** means the portion of a Cover Page that includes the key legal details and definitions for this Agreement that are not defined in the Standard Terms. The Key Terms may include details about Covered Claims, set the <span class="coverpage_link">Governing Law</span>, or contain other details about this Agreement.
+
+    15.18 **"Order Form"** means e.the portion of a Cover Page that includes the key business details and definitions for this Agreement that are not defined in the Standard Terms. An Order Form may include details about the level of access and use granted to the Cloud Service, nature and timing of <span class="orderform_link">Professional Services</span>, extent of <span class="orderform_link">Technical Support</span>, or other details about the Product.
+
+    15.19 **"Personal Data"** will have the meaning(s) set forth in the Applicable Data Protection Laws for personal information, personal data, personally identifiable information, or other similar term.
 
-    15.18 **"Personal Data"** will have the meaning(s) set forth in the Applicable Data Protection Laws for personal information, personal data, personally identifiable information, or other similar term.
+    15.20 **"Product”** means the Cloud Service, Software, and Documentation.
 
-    15.19 **"Protected Party"** means a party to this Agreement when the party is receiving the benefit of protection for a particular Covered Claim.
+    15.21 **"Protected Party"** means a party to this Agreement when the party is receiving the benefit of protection for a particular Covered Claim.
```

Comment:
I messed up the alphabetical order of definitions. It should be:

15.20 "Product"
15.21 "Prohibited Data"
15.22 "Protected Party"
15.23 "Recipient"
15.24 "Software"
15.25 "Usage Data"
15.26 "User"